prebid-mobile/pbm-api/ios/code-integration-ios.md

```diff
@@ -88,49 +88,68 @@ scripts/buildPrebidMobile.sh
 
 This will output the PrebidMobile.framework.
 
-## Initialize SDK
+## Add SDK
+
+### Set Prebid Server
 
 Once you have a [Prebid Server](/prebid-mobile/prebid-mobile-getting-started.html), you will add 'account' info to the Prebid Mobile. For example, if you're using the AppNexus Prebid Server:
 
 ```
-Prebid.shared.prebidServerAccountId = "YOUR_ACCOUNT_ID"
+Prebid.shared.prebidServerAccountId = YOUR_ACCOUNT_ID
```

Comment:
Account ID is a string

Reply:
`YOUR_ACCOUNT_ID` here is like a global or static constant. It is still a string, and publishers cannot set the prebidServerAccountId as any other data type. 

If it's not convenient, I'll restore the code snippets.

Reply:
Ok. Thanks.